src/Sarif.Sarifer/SpamBackgroundAnalyzer.cs

```diff
@@ -0,0 +1,163 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.IO;
+using System.Text.RegularExpressions;
+
+using Microsoft.CodeAnalysis.Sarif.Writers;
+
+namespace Microsoft.CodeAnalysis.Sarif.Sarifer
+{
+    [Export(typeof(IBackgroundAnalyzer))]
+    internal class SpamBackgroundAnalyzer : BackgroundAnalyzerBase, IBackgroundAnalyzer
```

Comment:
>SpamBackgroundAnalyzer [](start = 19, length = 22)

You know, this is so similar to the POC analyzer -- what do you think of the idea of just replacing the POC analyzer? Just renaming it to `SpamBackgroundAnalyzer`, and making the few small changes you made here? We don't really need both. The POC analyzer was just that -- a proof of concept. The concept is now proven :-)

Reply:
(I'm genuinely asking your opinion here.)

---
In reply to: [528043775](https://github.com/microsoft/sarif-visualstudio-extension/pull/275#discussion_r528043775) [](ancestors = 528043775)